modules/ossm-vs-istio.adoc

```diff
@@ -3,42 +3,27 @@ Module included in the following assemblies:
+[id="ossm-vs-istio_{context}"]
+= Differences between Istio and {ProductName} 
 
-{ProductName} installs a multi-tenant control plane by default. You specify the projects that can access the {ProductShortName}, and isolate the {ProductShortName} from other control plane instances.
+An installation of {ProductName} differs from an installation of Istio in multiple ways. The modifications to {ProductName} are sometimes necessary to resolve issues, provide additional features, or to handle differences when deploying on OpenShift.
 
-[id="ossm-mt-vs-clusterwide_{context}"]
-= Multi-tenancy in {ProductName} versus cluster-wide installations
+[id="ossm-cli-tool_{context}"]
+== Command line tool
 
-The main difference between a multi-tenant installation and a cluster-wide installation is the scope of privileges used by the control plane deployments, for example, Galley and Pilot. The components no longer use cluster-scoped Role Based Access Control (RBAC) resource `ClusterRoleBinding`, but rely on project-scoped `RoleBinding`.
+The Istio project provides the `istioctl` tool as a configuration command line utility that allows service operators to debug and diagnose their Istio service mesh deployments.
 
-Every project in the `members` list will have a `RoleBinding` for each service account associated with a control plane deployment and each control plane deployment will only watch those member projects. Each member project has a `maistra.io/member-of` label added to it, where the `member-of` value is the project containing the control plane installation.
-
-{ProductName} configures each member project to ensure network access between itself, the control plane, and other member projects. The exact configuration differs depending on how OpenShift software-defined networking (SDN) is configured. See About OpenShift SDN for additional details.
-
-If the {product-title} cluster is configured to use the SDN plug-in:
-
-* *`NetworkPolicy`*: {ProductName} creates a `NetworkPolicy` resource in each member project allowing ingress to all pods from the other members and the control plane. If you remove a member from {ProductShortName}, this `NetworkPolicy` resource is deleted from the project.
-+
-[NOTE]
-====
-This also restricts ingress to only member projects. If ingress from non-member projects is required, you need to create a `NetworkPolicy` to allow that traffic through.
-====
-
-* *Multitenant*: {ProductName} joins the `NetNamespace` for each member project to the `NetNamespace` of the control plane project (the equivalent of running `oc adm pod-network join-projects --to control-plane-project member-project`). If you remove a member from the {ProductShortName}, its `NetNamespace` is isolated from the control plane (the equivalent of running `oc adm pod-network isolate-projects member-project`).
-
-* *Subnet*: No additional configuration is performed.
+{ProductName} does not support the `istioctl` tool, instead it uses the `oc` command line tool.
```

Comment:
While it's true that we don't support `istioctl`, I'm not sure that I'd call oc our replacement. The overlapping functionality between the two is very small right now.

Reply:
I slightly tweaked the text here.